src/oneview-sdk/connection.js

```diff
@@ -28,26 +28,27 @@ import { isTerminal } from './tasks';
 //TODO this is global to the NodeJS process we probably want to figure out a cleaner way to do this
 process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
 
+var useProxy = false;
+
 export default class Connection {
-  constructor(applianceIp, apiVersion, readOnly){
+  constructor(oneview_config){
```

Comment:
This is a nit and I know it has always been named oneview_config but it would be nice if it could be changed to oneviewConfig.  Feels less like python.